spec.html

```diff
@@ -40907,7 +40906,7 @@ <h1>
         <h1>
           DetachArrayBuffer (
             _arrayBuffer_: an ArrayBuffer,
-            optional _key_: unknown,
+            optional _key_: an ECMAScript language value,
```

Comment:
I think this can be anything.

Reply:
Yeah, the spec isn't very clear on this point. I'll change it to `anything`.

Reply:
On second thought, `DetachArrayBuffer` passes `_key_` to `SameValue()`, which only accepts language values, which is presumably why I set `_key_`'s type to `an ECMAScript language value`. I'll leave it that way unless something else changes.

Reply:
@jmdyck Decision from editor call was to make the value as opaque as possible, so `any` would be better here. That will also necessitate the change from `SameValue` to `is` for comparison.